package.json

```diff
@@ -51,14 +51,15 @@
   },
   "scripts": {
     "postinstall": "yarn --cwd ./examples",
-    "prebuild": "rimraf dist && npm run typegen",
-    "build": "rollup -c && yarn copy && rimraf dist/__tests__",
+    "build": "rimraf dist && npm run typegen && rollup -c && yarn copy && rimraf dist/__tests__ && npm run replace-reconciler-imports",
```

Comment:
Mixing yarn with npm run?

Reply:
Woops. Force of habbit. I'll update these to use yarn.

Reply:
No no, npm run should be fine, I believe that yarn automatically detects npm calls

Reply:
Probably best for consistency though. I have another [PR](#830) that mentions running everything with `yarn`, and this would contradict that. 😂